### ISSUE TITLE
zwe only shows one missing config option at a time

### ISSUE THREAD

**Issue by dkelosky** [zowe/zowe-install-packaging#4023]
I had a `zwe config|install` error (I forget which) that gave:

`Error ZWEL0157E: Zowe Caching Service VSAM data set name (components.caching-service.storage.vsam.name) is not defined in Zowe YAML configuration file.`

However, on a previous run of the `zwe` I had already gotten: 

`Error ZWEL0157E: Zowe Caching Service VSAM data set Non-RLS volume (zowe.setup.vsam.volume) is not defined in Zowe YAML configuration file.`

Should the `zwe` installer surface multiple errors instead of one at a time so that I can address most errors on the first pass?

**Comment by balhar-jakub** [comment]
Yes, it should. There is even the zwe validate command that validates the zowe.yaml. This seems like a request for enhancement to run the zwe validate as the first phase of init and install to provide the information on all the issues we can see with the .yaml file.

**Comment by dkelosky** [comment]
I think it is.  Hopefully some one else with engage to confirm whether or not it's a good idea to pursue.

**Comment by Martin-Zeithaml** [comment]
* `zwe validate` is always run for each command, which can be executed via `configmgr`
  * `zwe install` and `zwe init` are not, but they will be run by configmgr (in V3, I hope)
* Schema validation will not help, this is valid zowe.yaml file, but I am afraid you would not be able to install zowe with it:
```
zowe:
  useConfigmgr: true
```
* `zwe` is script (shell or javascript), it will stop on the first error
  * But it would be possible to check all parameters in advance
  * I personally do prefer less code and possibly one error per execution
